packages/desktop-gui/src/settings/configuration.jsx

```diff
@@ -48,7 +48,28 @@ const normalizeWithoutMeta = (value = {}) => {
 }
 
 const ObjectLabel = ({ name, data, expanded, from, isNonenumerable }) => {
-  const formattedData = formatValue(data)
+  let formattedData = formatValue(data)
+
+  const iconStyles = {
+    marginLeft: '8px',
+  }
+
+  const editableInputStyles = {
+    height: '18px',
+    width: '100px',
+  }
+  const [isEditable, setIsEditable] = useState(false)
+
+  const [value, setValue] = useState('')
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    setIsEditable(!isEditable)
+
+    formattedData = value
```

Comment:
@jennifer-shehane I have tried directly changing the `formattedData`'s state here. However, it won't work. It's probably related to how to mutate state using `mobx` which I'm not familiar with plus there is probably an abstraction on top of it in this codebase. 

Can you point to where/how to change the state of the config values?